action.yml

```diff
@@ -0,0 +1,51 @@
+name: Create issue
+description: GitHub Action to create an issue as the github-actions user
+
+inputs:
+  title:
+    type: string
+    required: true
+  description:
+    type: string
+    required: true
+
+runs:
+  using: composite
+  steps:
+
+    - name: Create issue
+      shell: bash
+      env:
+        TITLE: ${{ inputs.title }}
+        DESCRIPTION: ${{ inputs.description }}
+        GITHUB_REPOSITORY: ${{ github.repository }}
+      run: |
+        # Escape double quotes '"' => '\"'
+        TITLE=${TITLE//\"/\\\"}
+        DESCRIPTION=${DESCRIPTION//\"/\\\"}
+
+        # The 41898282+ email prefixed is the required, matches the ID here
+        # https://api.github.com/users/github-actions%5Bbot%5D
+        # https://github.community/t/github-actions-bot-email-address/17204/6
+        # git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Why is this commented out? If it's not needed, just remove it altogether.

Reply:
Removed